Use UpdatePath instead of Commit when describing tree updates (#399)
mlswg/mls-protocol: draft-ietf-mls-protocol.md

## Ratchet Tree Evolution

> We now describe how members ratchet forward their group's shared secret with
> fresh key material. To do so, one member of the group will generate fresh key

Reviewer: ```suggestion
A member of an MLS group advances the key schedule to provide forward secrecy
and post-compromise security by providing the group with fresh key material to
be added into the group's shared secret. To do so, the member generates fresh key
```

... and change to present tense in the rest of the sentence

## Ratchet Tree Evolution

> A member of a MLS group advances the key schedule to provide forward secrecy

Reviewer: ```suggestion
A member of an MLS group advances the key schedule to provide forward secrecy
```

    A    B    C   D
~~~~~

After performing these operations, the generator of the UpdatePath MUST
delete the leaf_secret.

## Synchronizing Views of the Tree

After generating fresh key material and applying it to ratchet forward their
local tree state as described in the prior section, the generator must broadcast

> this update to other members of the group via an UpdatePath message, who

Reviewer: ```suggestion
this update to other members of the group in a Commit message, who
```

... or "by including an UpdatePath in a Commit message"
